### DISCUSSION TITLE
Do you think the result of tests are correct?

### DISCUSSION THREAD

**Discussion by ghost** [mcuee/libusb-win32#71] [Q&A] [answered]
I was attepted compile libnfc with libusb-win32, yet it couldn't find any devices which troubled me serveral days (I ensured that I plugged my FT232RL and PN532 module). After two days of debug, I began to suspect libusb-win32 make thing wrong on my machine. So I run the tests in libusb-win32 source tree, getting the following result.
```
$ ./testlibusb

Dev #0: 0000 - 0000
```
I would appreciate it if you could give me a hand.

**Comment by ghost** [comment]
I reconfirmed that I installed FTDI driver and libusb-win32 driver, here're the partial result of the `driverquery` command
```
FTDIBUS      USB Serial Converter D Kernel        2021/7/5 20:37:11
FTSER2K      USB Serial Port Driver Kernel        2021/7/5 20:37:48
libusb0      libusb-win32 - Kernel  Kernel        2024/2/22 11:52:25
```
I noticed #35, is the FT232R* chip works properly with libusb-win32 now?  If not, maybe I should purchase another one...

**Comment by ghost** [comment]
![屏幕截图 2024-06-18 125356](https://github.com/mcuee/libusb-win32/assets/91833768/bd7c0379-e9f5-4af5-a132-58c3083cadb8)
Here is the screenshot of testlibusb-win.exe

**Reply by mcuee** [reply, marked as answer]
You have not installed libusb-win32 driver yet.

1) Filter driver -- using the filter driver wizard, not recommended since it may not work with some devices.
2) Device driver -- recommended, please use Zadig to install the device driver, this will replace the FTDI driver and your device will no longer function as a USB serial port.

**Reply by ghost** [reply]
Thank you for your patience. libusb-win32 now works correctly! :)
